bundles/org.openhab.binding.airvisualnode/pom.xml

```diff
@@ -14,11 +14,17 @@
 
   <name>openHAB Add-ons :: Bundles :: AirVisual Node Air Quality Monitor Binding</name>
 
+  <properties>
+    <bnd.importpackage>
+      !jcifs
+    </bnd.importpackage>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.samba.jcifs</groupId>
       <artifactId>jcifs</artifactId>
-      <version>1.3.17</version>
+      <version>1.3.14-kohsuke-1</version>
```

Comment:
Not 100% sure, but IIRC there was a reason to chose 1.3.17. Maybe a security issue?

Reply:
My problem is that 1.3.17 does not seem to exist at all in any public repo...

Reply:
https://mvnrepository.com/artifact/jcifs/jcifs/1.3.17

Would this work?

Reply:
Thanks, that looks good indeed!